services/src/main/java/org/fao/geonet/api/records/MetadataWorkflowApi.java

```diff
@@ -473,6 +474,17 @@ public MetadataStatus convertParameter(int id, MetadataStatusParameter parameter
             }
             status.setTitle(title);
 
+            String uuid = uuids.get(s.getId().getMetadataId());
+            if (uuid == null) {
+                try {
+                    // Collect metadata uuid. For now we use Lucene
+                    uuid = LuceneSearcher.getMetadataFromIndexById(language, s.getId().getMetadataId() + "", "_uuid");
```

Comment:
I would 
```java
@Autowired
IMetadataUtils metadataUtils;
String uuid = metadataUtils.getMetadataUuid(Integer.toString(s.getId().getMetadataId()));
```
 to keep it similar to what is done in other places?

Reply:
Indeed.